precompiles/testutil/logs.go

```diff
@@ -52,6 +55,9 @@ func CheckLogs(logArgs LogCheckArgs) error {
 	availableEventIDs := make([]string, 0, len(ethRes.Logs))
 	for _, log := range ethRes.Logs {
 		availableEventIDs = append(availableEventIDs, log.Topics[0])
+		println("owner: ", log.Topics[1])
+		println("spender: ", log.Topics[2])
+		println("value: ", log.Data)
```

Comment:
will remove these print statements, they showcase the problem with ERC20 approvals using the MinterBurnerDecimals contract.